homeassistant/components/sensor/citybikes.py

```diff
@@ -13,7 +13,8 @@
 import voluptuous as vol
 
 import homeassistant.helpers.config_validation as cv
-from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.components.sensor import (
+    PLATFORM_SCHEMA, DOMAIN as PARENT_DOMAIN)
```

Comment:
Instead of importing as `PARENT_DOMAIN`, change the name of the existing `DOMAIN` constant in this module. You're not allowed to overwrite the component domain in the platform, in this case the sensor domain. Eg:
```py
CITYBIKES_DATA = 'citybikes'
```

Reply:
Fixed that.